src/common_types/mod.rs

```diff
@@ -42,3 +42,22 @@ pub struct BlockHeader {
     pub mix_hash: H256,
     pub nonce: u64,
 }
+
+#[derive(Clone, Debug, PartialEq)]
+pub struct BlockTransaction {
+    pub nonce: U256,
+    pub gas_price: U256,
+    pub gas_limit: U256,
+    pub to: H160,
+    pub value: U256,
+    pub input_data: Vec<u8>,
+    pub v: u8,
+    pub r: U256,
+    pub s: U256,
+}
+
+#[derive(Clone, Debug, PartialEq)]
+pub struct BlockBody {
+    pub transactions: Vec<BlockTransaction>,
+    pub ommers: Vec<BlockHeader>,
+}
```

Comment:
We will probably need to crate common_types lib or something like that so that other modules can share same structures. But for now this is fine.